yang-schema-comparison/draft-ietf-netmod-yang-schema-comparision.xml

```diff
@@ -33,6 +33,122 @@
+            a description of a leaf?</li>
+            <li>Metadata arguments which relies on the formatted input text.
+                E.g description, contact (etc), extension (how does the user
+                want to tune verbosity level for editorial changes: whitespace, spelling, editorial, potentially-nbc?</li>
+            <li>XPath, must, when: don't normalize XPath expressions</li>
+            <li>presence statements</li>
+        </ul>
+      </section>
+      <section>
+        <name>error-tags, error messages, and other error statements</name>
+        <t>Error tags and messages might be relied on verbatim by users.</t>
+        <ul>
+            <li>error-tag: standardized in <xref target="RFC6241"/></li>
+            <li>error-app-tag: arbitrary text (<xref target="RFC6241"/> but
+                also model)</li>
+            <li>error-message: arbitrary</li>
+        </ul>
+        <t>Failed must statement, error-message, assumed NBC</t>
+        <t>Default behaviour is changes to error tags, messages etc are NBC.</t>
+      </section>
+      <section>
+        <name>Comparison on module or full schema (YANG artifact, arbitrary blob.
+            Questions</name>
+          <ul>
+              <li>features</li>
+              <li>packages vs directories vs libraries vs artifact</li>
+              <li>package specific comparison, package metadata or only looking
+                  at the modules</li>
+              <li>import only or implemented module</li>
+          </ul>
+        <t>
+          Filter out comparison for a specific subrtree, path etc. Use case
+          for on-wire e.g. yang subscriptions, did the model change fro what is
+          subscribed on?
+      </t>
+      </section>
+    </section>
+    <section anchor="open-issues">
+      <name>Open Issues</name>
+      <t>{ This section is only to present the current ongoing work, not part of
+      the final draft. }</t>
+      <t>The following issues have not ben discussed in any wider extent yet.</t>
+      <section>
+          <name>Override/per-node tags</name>
+      </section>
+      <section>
+          <name>Separate rules for config vs state</name>
+      </section>
+      <section>
+          <name>Tool/report verbosity</name>
+            <ul>
+              <li>where to report changes (module, grouping, typedef, uses)</li>
+              <li>output level (conceptual level or exact strings)</li>
+              <li>granularity: error/warning/info level per reported change
+                  category</li>
+            </ul>
+      </section>
+      <section>
+          <name>sub-modules</name>
+      </section>
+      <section>
+          <name>Write algorithm in pseudo code or just describe the rules/goals
+              in text?</name>
+      </section>
+      <section>
+          <name>Categories in the report: bc, nbc, potentially-nbc, editorial.
+          Allow filtering in the draft without defining it?</name>
```

Comment:
One option might also be to show a full diff.  That is, _explain_ the why the tool arrived at the results.

Reply:
You mean presenting the category alongside the full diff?

Reply:
Yeah, like a `--details` option where the tool would "explain" why nbc was chosen as an example.

Reply:
Or just more --verbose levels maybe? More verbose args give more detailed
information about the reasoning about the decision, e.g. -vvv.

Reply:
Added text for this.